### ISSUE TITLE
Release the https:: fix

### ISSUE THREAD

**Issue by gimyboya** [zoomoid/strapi-provider-upload-aws-s3-advanced#17]
**Describe the bug**
I have upgraded to the latest version and still have the issue fixed by #15. I am guessing the PR didn't make it to the release. currently uploading files will store them with broken urls.

**To Reproduce**
1- use latest version
2- use the following env variables
```
AWS_ACCESS_KEY_ID=
AWS_ACCESS_SECRET=
AWS_REGION='us-west-004'
AWS_BUCKET=
CDN_BASE_URL=
AWS_ENDPOINT='https://s3.us-west-004.backblazeb2.com'
```
3- upload a file
4- get a broken url
**Expected behavior**

Urls should not start with `https:://` but with `https://`

 - Version 4.2.0

**Additional context**
Currently the latest release attached source code doesn't contain the fix

**Comment by zoomoid** [comment]
You are absolutely right, it never made its way to a release, but it's now online, thanks for reminding me!
